src/components/Notes/Notes.module.scss

```diff
@@ -43,7 +43,7 @@
 	}
 
 	.list {
-		display: grid;
+		display: flexbox;
```

Comment:
This kind of change will disrupt the whole Grid layout. It is not what we are looking for as a solution. I suggest you take a read on this CSS property https://developer.mozilla.org/en-US/docs/Web/CSS/word-break.